meta-aos-vm-common/recipes-aos/aos-servicemanager/aos-servicemanager_git.bbappend

```diff
@@ -43,6 +43,8 @@ python do_update_config() {
 
     data["CMServerURL"] = main_node_id+":8093"
 
+    data["JournalAlerts"] =  {"Filter": ["50-udev-default.rules"]}
```

Comment:
This is static configuration. Better to add it into the config file directly for main and remote node.